test api/REST Client/OData v4.http

```diff
@@ -0,0 +1,563 @@
+@clientid = 7e930f8e-fac4-43a6-a242-3a4079ac4553
```

Comment:
Also @clientId and tenant it's not neccesary to put a value here. You can just leave a comment like a //TODO Input client Id value. In this case it will be more easy for the user of this template to understand what he needs to change in order to make it work